apis/fluent-en_GB/atrium-api-fluent-en_GB/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableSubjectChangerSamples.kt

```diff
@@ -0,0 +1,45 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.asList
+import ch.tutteli.atrium.api.fluent.en_GB.toContain
+import ch.tutteli.atrium.api.fluent.en_GB.toEqual
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class IterableSubjectChangerSamples {
+    @Test
+    fun asListFeature() {
+        expect(0..2)
+            .asList()  // subject is now of type List<Int>
+            .toEqual(listOf(0, 1, 2))
+
+        fails {
+            expect(0..2)
+                .asList()  // subject is now of type List<Int>
+                .toContain(3)  // fails
+                .toContain(4)  // not evaluated/reported because above `toContain` already fails
+            //                    use `.asList { ... }` if you want that all expectations are evaluated
+        }
+    }
+
+    @Test
+    fun asList() {
+        expect(0..2)
+            // subject within this expectation-group is of type List<Int>
```

Comment:
This one is still on the wrong line. Did you forget to push your changes?

Reply:
> Move comment one line below

I saw such a comment, so I moved this comment one line below(Before chainging, this comment was on the line with `expect(0..2)`)

Reply:
I see, a misunderstanding. Put it after asList

Reply:
@robstoll 
Fixed this.